nrf52840/blinky/arduino/slow/slow.ino

```diff
@@ -0,0 +1,37 @@
+#include <Arduino.h>
+#include <Notecard.h>
+
+#define serialDebugOut Serial
+
+Notecard notecard;
+
+void setup()
+{
+    pinMode(LED_BUILTIN, OUTPUT);
+    Serial.begin(115200);
+
+    serialDebugOut.begin(115200);
+    notecard.setDebugOutputStream(serialDebugOut);
+
+    Wire.begin();
+    notecard.begin();
+
+    J *req = notecard.newRequest("hub.set");
+    JAddStringToObject(req, "mode", "continuous");
```

Comment:
If you are setting the `mode` to `continuous` to facilitate the Outboard DFU, then you will also want to set `sync` to `true`.
```suggestion
    JAddStringToObject(req, "mode", "continuous");
    JAddBoolToObject(req, "sync", true);
```